src/domain/user.js

```diff
@@ -171,4 +171,36 @@ export default class User {
 
     return foundUsers.map((user) => User.fromDb(user))
   }
+
+  static async _updateUser(id, json) {
```

Comment:
An underscore was an old way to inform developers that a function should be considered private, meaning only the class its defined on should use it. This function is being used outside of the class, so it should not be considered private. Let's remove the underscore